Cubical/HITs/SetTruncation/Properties.agda

```diff
@@ -56,3 +56,23 @@ elimSetTrunc3 : {B : (x y z : ∥ A ∥₀) → Type ℓ}
                 (x y z : ∥ A ∥₀) → B x y z
 elimSetTrunc3 Bset g = elimSetTrunc2 (λ _ _ → hLevelPi 2 λ _ → Bset _ _ _) (λ a b →
                        elimSetTrunc (λ _ → Bset _ _ _) (λ c → g a b c))
+
+
+setTruncIsSet : isSet ∥ A ∥₀
+setTruncIsSet a b p q = squash₀ a b p q
+
+setId : isSet A → A ≡ ∥ A ∥₀
```

Comment:
Why is the conclusion flipped compared to `propId`? I'm not sure which version is the easiest to use, but it would be good if they were the same